cupyx/scipy/special/_stats_distributions.py

```diff
@@ -37,7 +37,7 @@
 #define NPY_SQRT1_2   0.707106781186547524400844362104849039  /* 1/sqrt(2) */
 
 template <typename T>
```

Comment:
Now you can eliminate this template type.

Reply:
Fixed